seqr/views/utils/test_utils.py

```diff
@@ -577,3 +715,11 @@ def call_request_json(self, index=-1):
     'numExon': 2,
     '_sort': [1049045387],
 }
+
+GOOGLE_API_TOKEN_URL = 'https://oauth2.googleapis.com/token'
+GOOGLE_OAUTH2_URL = 'https://accounts.google.com/o/oauth2/'
+GOOGLE_AUTHORIZATION_URL = '{}auth'.format(GOOGLE_OAUTH2_URL)
+GOOGLE_ACCESS_TOKEN_URL = '{}token'.format(GOOGLE_OAUTH2_URL)
+GOOGLE_REVOKE_TOKEN_URL = '{}revoke'.format(GOOGLE_OAUTH2_URL)
```

Comment:
Do we need all of these now that we got rid of service accounts?

Reply:
No. Delete them.

Reply:
Actually, we still need these but we don't need the one (`GOOGLE_API_TOKEN_URL`) above